src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/resource/VisitResource.kt

```diff
@@ -0,0 +1,69 @@
+package uk.gov.justice.digital.hmpps.visitscheduler.resource
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.media.Content
+import io.swagger.v3.oas.annotations.media.Schema
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import org.springframework.http.MediaType
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.validation.annotation.Validated
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.visitscheduler.config.ErrorResponse
+import uk.gov.justice.digital.hmpps.visitscheduler.jpa.Visit
+import uk.gov.justice.digital.hmpps.visitscheduler.service.VisitSchedulerService
+import java.time.LocalDateTime
+import javax.validation.constraints.NotBlank
+
+@RestController
+@Validated
+@RequestMapping(name = "Visit Resource", path = ["/visits"], produces = [MediaType.APPLICATION_JSON_VALUE])
+class VisitResource(
+  private val visitSchedulerService: VisitSchedulerService
+) {
+
+  @PreAuthorize("hasRole('ROLE_PLACEHOLDER_VISIT')")
+  @GetMapping("/prisoner/{prisonerId}")
+  @Operation(
+    summary = "Get visits for prisoner",
+    description = "Retrieve all visits for a specified prisoner",
+    responses = [
+      ApiResponse(
```

Comment:
I think you can remove the 200 documentation, I found it caused an issue with list, but try it and see